gradle/gradle-mvn-push.gradle

```diff
@@ -57,7 +57,7 @@ afterEvaluate { project ->
                 }
 
                 pom.project {
-                    name POM_NAME
+                    name project.ext.artifactName
```

Comment:
Rather than update this file, I think we should put POM_NAME and POM_ARTIFACT_ID into `postman/gradle.properties` and `postman-processor/gradle.properties` as recommended here: https://github.com/chrisbanes/gradle-mvn-push.